controls/roles/manage-service/tasks/main.yml

```diff
@@ -34,9 +34,10 @@
     file:
       path: "{{ item.split(':') | first }}"
       state: directory
-      owner: "{{ stereum_service_configuration.user }}"
+      owner: "2000"
```

Comment:
Is there a reason for forcing "2000" instead of using `{{ stereum_service_configuration.user }}`? Grafana for example needs to run as root, otherwise it doesn't work properly.

Reply:
you right, there is a no need to change this line, since the service users are "2000". Now users given as parameter